### ISSUE TITLE
JAWS announce read only when focus is set on the search box

### ISSUE THREAD

**Issue by christosptr** [mr-bobz/talarts#68]
Reproduction Steps
Navigate to the search box with keyboard navigation 

Actual Result
When the focus is set on the search text area JAWS announce "Read only search region "
And announce the search multiple times,

This is confusing because when a text field is in read mode you are not suppose to be able to type in it

Expected
JAWS should announce Edit not Read and should announce the search only once. 

Remove the <span class="screen-reader-text">Search for:<span>

**Comment by christosptr** [comment]
Current HTML 

<form action="http://talarts.knowbility.org" class="search-form" method="get" role="search">
                <label>
                    <span class="screen-reader-text">Search for:</span>
                    <input type="search" title="Search for:" name="s" value="" placeholder="Search …" class="search-field">
                </label>
                <input type="submit" value="Search" class="search-submit">
            </form>

Suggested Solution 
<div role="search"><form action="http://talarts.knowbility.org" class="search-form" method="get" >
<label>
         <input type="search" name="s" value="" placeholder="Search …" class="search-field">
</label>
                <input type="submit" value="Search" class="search-submit">
            </form>
</div>

**Comment by christosptr** [comment]
verified
